### PR TITLE
Fixes issues with editing users #540

### PR DESCRIPTION
**Pull request by lukaskabrt**
* wrong action name was used
* validations rules weren't correct

### CODE REVIEW

File: src/OrchardCore.Modules/Orchard.Users/Controllers/AdminController.cs [hunk 1]
```diff
@@ -228,12 +228,14 @@ IUserService userService
 
             if (ModelState.IsValid)
             {
-                if ((await _userManager.FindByNameAsync(model.UserName))?.Id != currentUser.Id)
+                var userWithSameName = await _userManager.FindByNameAsync(model.UserName);
+                if ( userWithSameName != null && userWithSameName.Id != currentUser.Id)
```
Comment (sebastienros) [inline src/OrchardCore.Modules/Orchard.Users/Controllers/AdminController.cs]: np (which means 'nit picking'): remove the extra space after the parenthesis.

Comment (lukaskabrt) [inline src/OrchardCore.Modules/Orchard.Users/Controllers/AdminController.cs]: I agree ... coding standards are coding standards :-)

File: src/OrchardCore.Modules/Orchard.Users/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (sebastienros) [inline src/OrchardCore.Modules/Orchard.Users/Properties/launchSettings.json]: Oversight, remove this file

File: src/OrchardCore.Modules/Orchard.Users/Orchard.Users.csproj [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```
Comment (sebastienros) [inline src/OrchardCore.Modules/Orchard.Users/Orchard.Users.csproj]: Undo these changes, another PR is already doing it

File: src/Orchard.Cms.Web/Orchard.Cms.Web.csproj [hunk 4]
```diff
@@ -16,10 +16,24 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Content Include="Packages\Orchard.Users\Module.txt" />
```
Comment (sebastienros) [inline src/Orchard.Cms.Web/Orchard.Cms.Web.csproj]: Undo the changes in this file

Comment (lukaskabrt) [inline src/Orchard.Cms.Web/Orchard.Cms.Web.csproj]: Ooops, didn't mean to commit this changes, sorry.
